tools/scan-images.sh

```diff
@@ -54,6 +54,10 @@ for image in $images; do
           --severity HIGH,CRITICAL \
           --output image-scan-output/${filename}.json \
           --ignore-unfixed \
+          --db-repository ghcr.io/aquasecurity/trivy-db:2 \
+          --db-repository public.ecr.aws/aquasecurity/trivy-db \
+          --java-db-repository ghcr.io/aquasecurity/trivy-java-db \
```

Comment:
should this be `ghcr.io/aquasecurity/trivy-java-db:1`?

https://github.com/aquasecurity/trivy/commit/3562529ddfb26d301311ed450c192e17011353df#diff-39850860d29b4340dd9348be6ff8891c761b0bf742fb978480d5f6763c9142abR108

Reply:
good catch, thx